packages/@o3r/core/schematics/component/container/templates/container/__name__.spec.ts.template

```diff
@@ -20,8 +21,8 @@ class Mock<%= blockName %>PresComponent {}
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [<%= componentName %><% if (componentStructure === 'full') { %>, Mock<%= blockName %>PresComponent <% } %>],
-      imports: [BrowserModule],
+      <% if (!standalone) { %>declarations: [<%= componentName %>],
+      <% } %>imports: [<% if (standalone) { %><%= componentName %><% } else { %>BrowserModule<% } %><% if (componentStructure === 'full') { %>, Mock<%= blockName %>PresComponent <% } %>]
```

Comment:
I don;t think you need an else there. `BrowserModule` is needed in both cases, no ?